DEPS

```diff
@@ -375,20 +378,20 @@ deps = {
    Var('chromium_git') + '/external/gyp.git' + '@' + '4801a5331ae62da9769a327f11c4213d32fb0dad',
 
    # Headers for Vulkan 1.1
-   'src/third_party/vulkan':
-   Var('github_git') + '/KhronosGroup/Vulkan-Docs.git' + '@' + 'v1.1.91',
+   # 'src/third_party/vulkan':
+   # Var('github_git') + '/KhronosGroup/Vulkan-Docs.git' + '@' + 'v1.1.91',
 
    'src/third_party/swiftshader':
    Var('swiftshader_git') + '/SwiftShader.git' + '@' + '5d1e8540407c138f47028d64684f3da599430aa4',
 
-   'src/third_party/angle':
-   Var('github_git') + '/google/angle.git' + '@' + '06d194e2ae7b1d7e0eda0c0c911eff92dec7d3d1',
+   # 'src/third_party/angle':
+   # Var('github_git') + '/google/angle.git' + '@' + '06d194e2ae7b1d7e0eda0c0c911eff92dec7d3d1',
```

Comment:
Can we exclude `swiftshader` as well?

Reply:
Yes we can exclude `swiftshader` in build job. But `test` job in current azurepipeline needs it. :(
https://github.com/flutter-tizen/engine/blob/2be5f48c2c200de23b2c4ffb12bd90b7b2d5a8da/azure-pipelines.yml#L20